src/mono/wasi/sample/console/Makefile

```diff
@@ -1,7 +1,7 @@
 include ../../Makefile.variable
 include ../SampleMakefile.variable
 
-BIN_DIR=WasiConsoleApp/bin/$(CONFIG)/net7.0
+BIN_DIR=WasiConsoleApp/bin/Debug/net7.0
```

Comment:
Why is this being changed?

Reply:
Then we can build the sample on debug mode and we are able to run `make debug`.

Another approach would be separate runtimeconfig and config, because the way that was before if I pass CONFIG=Debug, it would search also for a runtime in debug mode.

Reply:
> Another approach would be separate runtimeconfig and config, because the way that was before if I pass CONFIG=Debug, it would search also for a runtime in debug mode.

That would be a good idea. We already do it for debugger tests.